lpa-submissions-e2e-tests/package.json

```diff
@@ -5,16 +5,16 @@
   "main": "index.js",
   "scripts": {
     "test": "npx cypress run -e TAGS='not @wip'",
-    "test:e2e": "npx cypress run -e TAGS='not @wip'",
+    "test:e2e": "npx cypress run --headed -b chrome --config baseUrl=https://tim:password@lpa-questions-dev.planninginspectorate.gov.uk",
```

Comment:
i'm not sure this is a change you want to be making- specifically the pointing towards the dev environment.

I would expect this to be the hook we call during CI, and CI expects to run against it's own copy of the app running on localhost:9000.. pointing this towards the dev environment doesn't really make sense- the CI is going to run -before- anything has been deployed to dev, so unless i'm missing some other wiring: this will just never pass for new work.

secondly- not good to be checking the user-name+password into the (open to the public) repository- if you -do- need something that runs under github actions that needs username/password stuff we can talk about how to do it securely

Reply:
ok, I will make changes, though lpa runs on localhost:9001